src/ocean/text/convert/Float.d

```diff
@@ -94,6 +94,30 @@ private enum
 
 /******************************************************************************
 
+  Defines information needed to format Commas and decimals.
+
+ ******************************************************************************/
+
+public struct DecimalMark
+{
+    /// Character used for a comma
+    char  comma_char;
+    /// Character used for a decimal
```

Comment:
The word "decimal" here is a bit ambiguous. It could mean a decimal point or a number in decimal base.

I'd suggest calling it `decimal_mark_char` (https://en.wikipedia.org/wiki/Decimal_mark).